pkg/synchromanager/clustersynchro/resource_negotiator.go

```diff
@@ -21,6 +21,12 @@ import (
 	clusterpediafeature "github.com/clusterpedia-io/clusterpedia/pkg/utils/feature"
 )
 
+// excludedInternalResources is a list of internal resources that are not synchronized
+// if the feature gate SkipSyncLeases is enabled.
+var excludedInternalResources = sets.NewString(
+	"leases.coordination.k8s.io",
+)
```

Comment:
We've made feature gates specific to the leases resource, so I don't think we need to maintain the `excludedInternalResources` here, we could add it if we need it more in the future